charts/everest/templates/everest-olm/packageserver.apiservice.yaml

```diff
@@ -0,0 +1,35 @@
+{{- $tls := fromYaml (include "olm.certs" .) }}
+---
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1.packages.operators.coreos.com
```

Comment:
This chart changes the way OLM is deployed. Earlier we applied a CSV for the packageserver component, but this lead to many issues. For example, when running `helm uninstall ...` the OLM operator always got deleted first, and the CSV finalizer was stuck. As a result, the entire `everest-olm` namespace got stuck.

By deploying the components manually (using native objects such Deployment, APIService, etc.) helm handles deletion correctly..